### PR TITLE
Explorer add error boundaries

### PR DESCRIPTION
**Pull request by Jibz1**

### CODE REVIEW

File: apps/explorer/src/pages/object-result/views/ObjectView.tsx [hunk 1]
```diff
@@ -24,7 +25,13 @@ function ObjectView({ data }: { data: DataType }) {
                     subtitle={name}
                 />
             </div>
-            {isPackage ? <PkgView data={data} /> : <TokenView data={data} />}
+            <ErrorBoundary>
+                {isPackage ? (
+                    <PkgView data={data} />
+                ) : (
+                    <TokenView data={data} />
+                )}{' '}
```
Comment (Jordan-Mysten) [inline apps/explorer/src/pages/object-result/views/ObjectView.tsx:33]: Looks like it picked up an extra space

File: apps/explorer/src/pages/object-result/ObjectResult.tsx [hunk 2]
```diff
@@ -93,7 +94,11 @@ function ObjectResultAPI({ objID }: { objID: string }) {
     }, [objID, rpc]);
 
     if (showObjectState.loadState === 'loaded') {
-        return <ObjectView data={showObjectState as DataType} />;
+        return (
+            <ErrorBoundary>
```
Comment (Jordan-Mysten) [inline apps/explorer/src/pages/object-result/ObjectResult.tsx:98]: I feel like this boundary more logically exists in the parent

File: apps/explorer/src/components/ownedobjects/views/OwnedObjectView.tsx [hunk 3]
```diff
@@ -37,7 +38,9 @@ export default function OwnedObjectView({ results }: { results: DataType }) {
                     <div className={styles.ownedobjectheader}>
                         <h2>Coins</h2>
                     </div>
-                    <OwnedCoinView results={coin_results} />
+                    <ErrorBoundary>
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/ownedobjects/views/OwnedObjectView.tsx:41]: Feels like something that more logically should exist in the composing page

**Review by Jordan-Mysten — APPROVED** [review]
Few small things but nothing blocking

File: apps/explorer/src/components/ownedobjects/views/OwnedObjectView.tsx (hunk 3 above)
Comment (Jibz1) [reply apps/explorer/src/components/ownedobjects/views/OwnedObjectView.tsx:41]: Done

File: apps/explorer/src/pages/object-result/ObjectResult.tsx (hunk 2 above)
Comment (Jibz1) [reply apps/explorer/src/pages/object-result/ObjectResult.tsx]: done

File: apps/explorer/src/pages/object-result/views/ObjectView.tsx (hunk 1 above)
Comment (Jibz1) [reply apps/explorer/src/pages/object-result/views/ObjectView.tsx]: done

**Pull request merged**